docs/get-started/deployment-and-usage.md

```diff
@@ -263,6 +263,29 @@ nfd-master args, in which case nfd-master verifies that the NodeName presented
 by nfd-worker matches the Common Name (CN) of its certificate. This means that
 each nfd-worker requires a individual node-specific TLS certificate.
 
+#### Automated TLS certificate management using cert-manager
+
+[cert-manager](https://cert-manager.io/) can be used to automate certificate
+management between nfd-master and the nfd-worker pods. The instructions below describe
+steps how to set up cert-manager's [CA Issuer](https://cert-manager.io/docs/configuration/ca/) to
+sign `Certificate` requests for NFD components in `node-feature-discovery` namespace.
+
+1. Deploy cert-manager: `$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.0.3/cert-manager.yaml`
```

Comment:
We should prolly change to the latest version of cert-manager here

Reply:
to reduce maintenance burden this could also be something like `v{CERT_MANAGER_VERSION}`?

Reply:
I like the kind of instructions that you can just copy-paste into shell without visiting e.g. cert-manager site to figure out what could possibly be the latest compatible version